src/js/base/post-load-hooks.js

```diff
@@ -0,0 +1,136 @@
+define("pyret-base/js/post-load-hooks", function() {
+  function makeDefaultPostLoadHooks(runtime, main) {
```

Comment:
In general, I bet this second argument is going to need to be a `runtimeOptions` datatype of some sort.

Reply:
Yeah I had thought of that, but didn't notice the other points of variation, so didn't think it was warranted...